docs/_static/versioning.js

```diff
@@ -1,7 +1,7 @@
+const urlRoot = DOCUMENTATION_OPTIONS.URL_ROOT ?? "";
```

Comment:
Why does this value not exist sometimes?

Reply:
i dont know, thats the better question :\ i was trying to look through changelogs on sphinx7/8 but im not finding anything to point why it all of a sudden broke some time ago for us. i can keep poking

Reply:
Ah. Well, let's keep it for now :woman_shrugging: